### PR TITLE
session 1 changes

### PR DESCRIPTION
**Pull request by iswarya15**

### CODE REVIEW

File: hello-world/src/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Why is react known as React?
```
Comment (nitishnivedan) [inline hello-world/src/README.md:1]: good way of doing theoretical assignment, both coding and theory can be kept tracked of.

File: hello-world/src/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (nitishnivedan) [inline hello-world/src/index.html:1]: code formatting is not right. you can install prettier in your vs code extension. code seems to be fine

File: hello-world/src/README.md (hunk 1 above)
Comment (shaksham08) [reply hello-world/src/README.md:1]: true

**Review by shaksham08 — APPROVED** [review]
Looks good , I guess you missed adding element using simple HTML and vanilla js as well

File: hello-world/src/index.html (hunk 2 above)
Comment (iswarya15) [reply hello-world/src/index.html:1]: Thanks for the suggestion. Fixed formatting

**Comment by iswarya15** [comment]
> Looks good , I guess you missed adding element using simple HTML and vanilla js as well

Thanks. Updated suggested changes in the latest commit.

**Pull request merged**